#
# (c) Copyright 2015-2016 River Systems, all rights reserved.
#
# This is proprietary software, unauthorized distribution is not permitted.
#
string id
time start_time
time end_time
srslib_framework/TimingDataSample[] samples
